src/code/audio_load.c

```diff
@@ -800,15 +801,15 @@ AudioTable* AudioLoad_GetLoadTable(s32 tableType) {
  * @param sampleBankReloc information on the sampleBank containing raw audio samples
  */
 void AudioLoad_RelocateFont(s32 fontId, SoundFontData* fontDataStartAddr, SampleBankRelocInfo* sampleBankReloc) {
-    u32 soundOffset;     // Relative offset from the beginning of fontData directly to the sound/envelope
+    u32 soundOffset;     // Relative offset from the beginning of fontData directly to the tunedSample/envelope
     u32 soundListOffset; // Relative offset from the beginning of fontData to the list of soundOffsets/sfxs
```

Comment:
not sure why I didn't catch this earlier but `soundOffset` can also be ram `soundOffset = RELOC_TO_RAM(soundOffset);`

and both of these are also used for drums

NotLikeThis